lib/ansible/modules/cloud/amazon/dms_replication_subnet_group.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/python
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: dms_replication_subnet_group
+short_description: creates or destroys a data migration services subnet group
+description:
+    - Creates or destroys a data migration services subnet group
+version_added: "2.9"
+options:
+    state:
+        description:
+            - State of the subnet group
+        default: present
+        choices: ['present', 'absent']
+        type: str
+    identifier:
+        description:
+            - The name for the replication subnet group.
+              This value is stored as a lowercase string.
+              Must contain no more than 255 alphanumeric characters,
+              periods, spaces, underscores, or hyphens. Must not be "default".
+        type: str
+    description:
+        description:
+            - The description for the subnet group.
+        type: str
+    subnetids:
```

Comment:
```suggestion
    subnet_ids:
```
Snake casing would help with readability of this option.